Stylesheets/reset.css

```diff
@@ -0,0 +1,69 @@
+*,
+::before,
+::after {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html,
+body {
+  scroll-behavior: smooth;
+  font-family: "Montserrat", sans-serif;
+  line-height: 1.5rem;
+  color: var(--text-color);
+  background-color: var(--bg-dark);
+}
+
+html::-webkit-scrollbar {
+  display: none;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin: 0;
+}
+
+h1 {
+  font-size: 2rem;
+}
+
+h2 {
+  font-size: 1.5rem;
+}
+
+h3 {
+  font-size: 1rem;
+}
+
+li {
+  list-style: none;
+  margin: 1.5rem auto;
+}
+
+p {
+  margin-top: 1rem;
+  text-align: justify;
+}
+
+a {
+  text-decoration: none;
+  color: var(--primary-dark);
+  outline: none;
+}
+
+ul {
+  list-style-type: none;
+}
+
+button {
+  cursor: pointer;
+  text-align: center;
+  outline: none;
```

Comment:
Its not a good practice to make Outline of a button as none as it. It provides visual feedback for links that have "focus" when navigating a web document using the TAB key (or equivalent). This is especially useful for folks who can't use a mouse or have a visual impairment. If you remove the outline you are making your site inaccessible for these people